src/man/sssd.8.xml

```diff
@@ -90,6 +90,41 @@
                         log files are stored in <filename>/var/log/sssd</filename> and
                         there are separate log files for every SSSD service and domain.
                     </para>
+                    <para>
+                        This option is deprecated. It is replaced by
+                        <option>--logger=files</option>.
+                    </para>
+                </listitem>
+            </varlistentry>
+            <varlistentry>
+                <term>
+                    <option>--logger=</option><replaceable>value</replaceable>
+                </term>
+                <listitem>
+                    <para>
+                        Send messages to different place. This option overrides
+                        the value of the deprecated option
+                        <option>--debug-to-files</option>. The deprecated
+                        option will still work if the <option>--logger</option>
+                        is not used.
+                    </para>
+                    <para>
+                        <emphasis>stderr</emphasis>: Redirect debug messages to
+                        standard error output.
+                    </para>
+                    <para>
+                        <emphasis>files</emphasis>: Redirect debug messages to
+                        the log files. By default, the log files are stored in
+                        <filename>/var/log/sssd</filename> and there are
+                        separate log files for every SSSD service and domain.
+                    </para>
+                    <para>
+                        <emphasis>journald</emphasis>: Redirect debug messages
+                        to systemd-journald
+                    </para>
+                    <para>
+                        Default: not set
```

Comment:
I am just wondering why the Default is not 'files' here?

Reply:
Default value is `not set` due to be backward compatible. And you would not be able to override value with old flags `--debug-to-files` `--debug-to-stderr` (which is implicitly used in case of `sssd --interactive`)

Reply:
Okay, thanks for the explanation.